custom-elements/CustomElementRegistry.html

```diff
@@ -623,6 +623,18 @@
     assert_equals(customElements.get('existing-custom-element'), ExistingCustomElement);
 }, 'customElements.get return the constructor of the entry with the given name when there is a matching entry.');
 
+test(function () {
```

Comment:
We should probably add these test cases as well:

test(function () {
  assert_equals(customElements.getName(undefined), undefined);
}, 'customElements.getName must return undefined when called with undefined');

test(function () {
  assert_equals(customElements.getName(null), undefined);
}, 'customElements.getName must return undefined when called with null');

test(function () {
  assert_equals(customElements.getName(''), undefined);
}, 'customElements.getName must return undefined when called with a string');